partner_compassion/models/partner_compassion.py

```diff
@@ -183,6 +187,29 @@ def name_search(self, name, args=None, operator='ilike', limit=80):
         return self.browse(to_ret_ids).name_get()
 
     ##########################################################################
+    #                             ONCHANGE METHODS                           #
+    ##########################################################################
+
+    @api.onchange('lastname', 'birthdate_date', 'firstname')
+    def _onchange_partner(self):
+        db_info = self.search([('firstname', '=', self.firstname),
+                               ('lastname', '=', self.lastname),
+                               ('birthdate_date', '=', self.birthdate_date)])
+        list_hom = []
+        for inf in db_info:
+            list_hom.append((4, inf.id))
+        self.listehomonyme = list_hom
+        self.homonymeformat = list_hom
+        if db_info:
+            return {
+                     'warning': {'title': "Check-Double", 'message': "The "
+                                 "partner you want to dd may already be in "
```

Comment:
Change message to : `_('The partner you want to add may already exist. Please use the "Check duplicates" button to review it.')`